### ISSUE TITLE
Unable to generate a workflow id from the Onfido dashboard

### ISSUE THREAD

**Issue by Wes1324** [onfido/flutter-sdk#5]
### What was the expected behaviour?

In order to run the example Flutter app's workflow demo, I can go to my Onfido dashboard and create a workflow id.

### What happened instead?

I cannot see any area of the Onfido dashboard that allows me to create a workflow id, despite [this section](https://documentation.onfido.com/#create-workflow-run) of the Onfido docs stating that 'A Workflow must exist and be activated before you can create a Workflow Run. You can create and activate your Workflow in your Studio via Onfido Dashboard.'

### Version info:

- Onfido SDK Version: onfido_sdk: ^1.0.0
- iOS or Android: iOS
- Device/Simulator: Simulator

### Integration configuration:

```
API_TOKEN=<My Onfido Api token>
WORKFLOW_ID=<How do I get this?>
```

### Steps to reproduce:

I have created an API token from my Onfido dashboard and pasted that into the `.env` file. That API token seems to be working fine because the `createApplicant` method is successfully returning an `Applicant` with an id generated by Onfido. However, I cannot see anywhere in the Onfido dashboard where I can create a workflow id to put into my `.env` file. The lack of workflow id means that the `getWorkflowRunId` method throws and I keep seeing an error dialog saying 'Error Exception: Failed to create workflow run id'.

Please explain in more detail how I can generate a workflow id in order to run the example Flutter app? 

Additionally, I want to highlight that the API endpoint hit on line 33 of the `onfido_api.dart` file is `/v4/workflow_runs`, but attempting to hit this endpoint results in a 404 not found error. The v4 api doesn't seem to exist yet. The online Onfido docs only document up to api version 3.5.

**Comment by Wes1324** [comment]
Here is a screenshot of the Onfido dashboard that we see, with no option to generate a workflow id:

![image (7)](https://user-images.githubusercontent.com/25579138/202176063-4fff63bd-6b2d-4bd4-9f1d-0b536fce4158.png)

**Comment by alistairholmes** [comment]
Hi there. were you able to get the Workflow ID from the dashboard?

**Comment by Wes1324** [comment]
Nope, still haven't seen where in the dashboard I'm supposed to get it from. Are you having the same issue?

**Comment by alistairholmes** [comment]
Yep, also can't seem to find it.
